vic/drivers/shared_image/src/get_global_domain.c

```diff
@@ -65,10 +65,12 @@ get_global_domain(char          *nc_name,
                      run);
 
     for (i = 0; i < global_domain->ncells_total; i++) {
-        if (run[i]) {
+        if (run[i] == 1) {
```

Comment:
Maybe replace `1` with `DO_RUN` or something to allow for different run values. If that's a good idea open a separate issue for it, because no need to hold up the PR.